pwa/src/state/models/patientInfoForm.model.ts

```diff
@@ -2,33 +2,35 @@ import { Action, action } from 'easy-peasy'
 import { SEX, YN } from '../enums'
 
 interface PatientInfoFormFields {
-    firstName?: string
-    lastName?: string
+    first_name?: string
+    last_name?: string
     nickname?: string
-    sex?: SEX
-    birthDate?: Date
+    gender?: SEX
+    birthdate?: Date
     age?: number
     phone?: string
     town?: string
-    firstVisit?: YN
-    pregnant?: YN
-    G?: number
-    P?: number
-    A?: number
-    LMP?: Date
-    wantsPlanning?: YN
-    weight?: number
-    height?: number
-    bloodPressure?: string
-    temperature?: number
-    pulse?: number
+    first_visit?: YN
+    ob_pregnant?: YN
+    ob_gravida?: number
+    ob_para?: number
+    ob_abortus?: number
+    ob_last_menstrual_period?: Date
+    ob_wants_planning?: YN
+    weight_in_lbs?: number
+    height_in_inches?: number
+    bp_systolic_blood_pressure?: number
+    bp_diastolic_blood_pressure?: number
+    temp_in_fahrenheit?: number
+    pulse_rate?: number
+    // TODO: Can't find zScore proper name?
```

Comment:
Is there a spike/issue for this so we don't loose track of it

Reply:
There is now: #105